corehq/apps/app_manager/models.py

```diff
@@ -4859,6 +4860,13 @@ def wrap(cls, data):
                            if module.get('doc_type') != 'CareplanModule']
         self = super(Application, cls).wrap(data)
 
+        translations = data.get('translations')
+        labels = {lang: translations.get(lang).get('case.search.title') for lang in translations}
```

Comment:
You can use `items` to avoid needing to access `translations` in the start of the comprehension and be a little more concise: `{lang: stuff.get('case.search.title') for lang, stuff in translations.items()}` ("stuff" is a poor variable name but i can't think of a good one right now).

I think later tickets will be easier if this only populates `labels` with real translations. As is, it will also populate it with `None` for any language that exists in `translations` but doesn't have a value for `case.search.title`. Eventually there will be a default translation here, and we won't want None values to override it.

Reply:
so would this work? 
```python3
label_dict = {lang: label.get('case.search.title') 
            for lang, label in translations.items() if label}
```

Reply:
yep, that looks good

Reply:
great added the changes